client/src/components/Workbench/Workbench.jsx

```diff
@@ -17,6 +17,7 @@ import * as ROUTES from '../../constants/routes';
 import NotificationToast from '../Toasts';
 import OnboardingAdmin from '../Onboarding/OnboardingAdmin';
 import './Workbench.scss';
+import '../../index.css'
```

Comment:
Agree with Wes, let's include that comment. Also a semicolon for linting
```suggestion
import '../../index.css';
```